README.md

```diff
@@ -5,19 +5,13 @@
 - Ruby
 
```

Comment:
- [OPTIONAL] Please it will be nice to add a short description about your project as this will help users to quickly get an idea of what your project is about.